plugins/plugin-filter-antd/README.md

```diff
@@ -0,0 +1,35 @@
+## @superset-ui/plugin-filter-antd
+
+[![Version](https://img.shields.io/npm/v/@superset-ui/plugin-filter-antd.svg?style=flat-square)](https://www.npmjs.com/package/@superset-ui/plugin-filter-antd)
+
+This plugin provides native filter plugins based on AntD.
+
+### Usage
+
+Configure `key`, which can be any `string`, and register the plugin. This `key` will be used to lookup this chart throughout the app.
+
+Below is an example of how to use the Select filter plugin.
```

Comment:
Since these are a bit of a special case, I wonder if it's worth adding a sentence or two about why they exist

Reply:
Good idea, I'll do that. Aaaand while I'm at it, add a storybook entry.